Lib/multiprocessing/popen_spawn_win32.py

```diff
@@ -18,6 +19,7 @@
 TERMINATE = 0x10000
 WINEXE = (sys.platform == 'win32' and getattr(sys, 'frozen', False))
 WINSERVICE = sys.executable.lower().endswith("pythonservice.exe")
+STARTF_FORCEOFFFEEDBACK = 0x80
```

Comment:
I suggest updating "Modules/\_winapi.c" to define all of the `STARTF_*` constants.

```c
// winbase.h limits the STARTF_* flags to the desktop API as of 10.0.19041.
#ifndef STARTF_USESHOWWINDOW
#define STARTF_USESHOWWINDOW 0x00000001
#endif
#ifndef STARTF_USESIZE
#define STARTF_USESIZE 0x00000002
#endif
#ifndef STARTF_USEPOSITION
#define STARTF_USEPOSITION 0x00000004
#endif
#ifndef STARTF_USECOUNTCHARS
#define STARTF_USECOUNTCHARS 0x00000008
#endif
#ifndef STARTF_USEFILLATTRIBUTE
#define STARTF_USEFILLATTRIBUTE 0x00000010
#endif
#ifndef STARTF_RUNFULLSCREEN
#define STARTF_RUNFULLSCREEN 0x00000020
#endif
#ifndef STARTF_FORCEONFEEDBACK
#define STARTF_FORCEONFEEDBACK 0x00000040
#endif
#ifndef STARTF_FORCEOFFFEEDBACK
#define STARTF_FORCEOFFFEEDBACK 0x00000080
#endif
#ifndef STARTF_USESTDHANDLES
#define STARTF_USESTDHANDLES 0x00000100
#endif
#ifndef STARTF_USEHOTKEY
#define STARTF_USEHOTKEY 0x00000200
#endif
#ifndef STARTF_TITLEISLINKNAME
#define STARTF_TITLEISLINKNAME 0x00000800
#endif
#ifndef STARTF_TITLEISAPPID
#define STARTF_TITLEISAPPID 0x00001000
#endif
#ifndef STARTF_PREVENTPINNING
#define STARTF_PREVENTPINNING 0x00002000
#endif
#ifndef STARTF_UNTRUSTEDSOURCE
#define STARTF_UNTRUSTEDSOURCE 0x00008000
#endif
```

```c
    WINAPI_CONSTANT(F_DWORD, STARTF_USESHOWWINDOW);
    WINAPI_CONSTANT(F_DWORD, STARTF_USESIZE);
    WINAPI_CONSTANT(F_DWORD, STARTF_USEPOSITION);
    WINAPI_CONSTANT(F_DWORD, STARTF_USECOUNTCHARS);
    WINAPI_CONSTANT(F_DWORD, STARTF_USEFILLATTRIBUTE);
    WINAPI_CONSTANT(F_DWORD, STARTF_RUNFULLSCREEN);
    WINAPI_CONSTANT(F_DWORD, STARTF_FORCEONFEEDBACK);
    WINAPI_CONSTANT(F_DWORD, STARTF_FORCEOFFFEEDBACK);
    WINAPI_CONSTANT(F_DWORD, STARTF_USESTDHANDLES);
    WINAPI_CONSTANT(F_DWORD, STARTF_USEHOTKEY);
    WINAPI_CONSTANT(F_DWORD, STARTF_TITLEISLINKNAME);
    WINAPI_CONSTANT(F_DWORD, STARTF_TITLEISAPPID);
    WINAPI_CONSTANT(F_DWORD, STARTF_PREVENTPINNING);
    WINAPI_CONSTANT(F_DWORD, STARTF_UNTRUSTEDSOURCE);
```

Reply:
`STARTF_FORCEONFEEDBACK` and `STARTF_FORCEOFFFEEDBACK` could be generally useful for GUI scripts. We could add imports for them from `_winapi` in "Lib/subprocess.py", and document them in the "Windows Constants" section of "Doc/library/subprocess.rst". Then this code would import `STARTF_FORCEOFFFEEDBACK` from `subprocess`. Otherwise import it directly from `_winapi`.

Reply:
Good point, the constant probably belongs in `_winapi`. I'll move it there and rebase the commit. But maybe exposing all the constants from `subprocess` is a separate issue?

Reply:
> But maybe exposing all the constants from `subprocess` is a separate issue?

Definitely not all of them. I only suggested the possibility of exposing and documenting `STARTF_FORCEONFEEDBACK` and `STARTF_FORCEOFFFEEDBACK`. I'd rather add the flags to `subprocess` on a case by case basis, driven by real use cases.

Reply:
It's pretty much zero cost with no long-term support cost to just add all of the flags to `_winapi` now. It's a private extension module for internal use. We're free to add and remove anything there whenever we want, and the standard library is free to use anything from `_winapi` without documenting it.

Reply:
Oh, right, I see now what you were saying. I have added the flags to `subprocess` as well. (Seems like a good idea, since anyone launching background worker processes directly using `subprocess` would run into the same issue.)